lib/main_common.pm

```diff
@@ -8,7 +8,7 @@
 # without any warranty.
 
 # Summary: Attempt to merge common parts of sle/main.pm and opensuse/main.pm
-# Maintainer: Anton Smorodskyi<asmorodskyi@suse.com>
+# Maintainer:  qe-core@suse.de
```

Comment:
delete the leading space, only one space for separation after colon

Reply:
done